qa/common/gen_qa_model_repository

```diff
@@ -525,3 +523,66 @@ if [ $? -ne 0 ]; then
     echo -e "Failed"
     exit 1
 fi
+
+else # [[ "aarch64" != $(uname -m) ]]
```

Comment:
Just to check my understanding. We only generate `tensorrt` models on aarch64 with this change? Will we use the other backend models created on x86_64 when testing on aarch64?

Reply:
Yes it's correct observation for now it's only TensorRT models